### PR TITLE
refactor: upgrade node standards

### PR DESCRIPTION
**Pull request by pevisscher**

### CODE REVIEW

**Comment by pevisscher** [comment]
Current dependencies on/for this PR:
* main
  * **PR #61** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/61" target="_blank">Graphite</a>  👈
    * **PR #62** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/62" target="_blank">Graphite</a> 
      * **PR #63** <a href="https://app.graphite.dev/github/pr/skyleague/therefore/63" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/61?utm_source=stack-comment).

**Review by MickVanDuijn — APPROVED** [review]

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/61?utm_source=stack-comment).


Job Reference: JzO6cwGTfENaZFy5T9Os

**Comment by pevisscher** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/skyleague/therefore/61?utm_source=stack-comment).


Job Reference: ndSaF8BbFfaLQTchrjOV

**Pull request merged**
